modules/installation-user-infra-machines-iso.adoc

```diff
@@ -17,33 +17,30 @@ the machines that you create can access.
 
 .Procedure
 
-////
-This will work for GA.
-. Obtain the {op-system} images from the
-link:https://access.redhat.com/downloads[Product Downloads] page on the Red
-Hat customer portal.
-////
-
 . Upload the control plane, compute, and bootstrap Ignition config files that the
 installation program created to your HTTP server. Note the URLs of these files.
 
-
 . Obtain the {op-system} images that are required for your preferred method
 of installing operating system instances from the
 link:https://access.redhat.com/downloads[Product Downloads] page on the Red
 Hat customer portal or the
 link:https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.1/[{op-system} image mirror]
 page.
 +
+[IMPORTANT]
+====
+The {op-system} images might not change with every release of {product-title}.
+You must download images with the same version number, and that version must be
```

Comment:
I think I know what you mean by "You must download images with the same version number" (for the things you're downloading, all should have the same number) however I think that will be confusing in juxtaposition with the next bit. Also they're looking in a versioned directory, and they're only going to download 1 or 2 things so I don't think they'll get confused about the versions of those things matching each other. Best not to say anything?
```suggestion
You must download images with the highest version that is
```

Reply:
@sosiouxme, @wking, thanks! Is this acceptable?